### ISSUE TITLE
Collabora Online settings screen does not load

### ISSUE THREAD

**Issue by timar** [nextcloud/richdocuments#869]
**Describe the bug**
Browser console log:
Refused to execute script from 'https://staging.eu.collaboraonline.com/nc-richdocuments-next/index.php/apps/files/' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
﻿
**To Reproduce**
I wanted to try richdocuments from git / master branch. 

1. I installed Nextcloud 18
2. I installed Collabora Online app
3. I cloned the richdocuments repo to apps/richdocuments
4. See error (the settings screen does not load)

**Comment by juliushaertl** [comment]
Yes, installing from git also requires some additional build steps in order to transpile the javascript code. Make sure you have a recent version of node/npm installed and run `npm ci` to install the dependencies and `npm run dev` to build the javascript bundles.

I guess we should add some kind of warning to the settings page, when the bundles are not build yet.

**Comment by timar** [comment]
Thanks @juliushaertl , it works for me now.

**Comment by juliushaertl** [comment]
Ok, let's keep this as an enhancement open to add a small notice to the admin settings.

**Comment by juliusknorr** [comment]
We don't do this with any app and it is kind of standard these days in Nextcloud apps and documented.

Lets close this, logs should tell otherwise that the js file is not found.
